blocks/data-cards/data-cards.js

```diff
@@ -0,0 +1,67 @@
+export default function decorate(block) {
+    /* change to ul, li */
+    const ul = document.createElement('ul');
+    ul.classList.add('data-cards')
+    const paragraphs = block.querySelectorAll('p');
+    [...block.children].forEach((row) => {
+        const li = document.createElement('li');
+        li.classList.add('data-card')
+        while (row.firstElementChild) li.append(row.firstElementChild);
+        [...li.children].forEach((div, index) => {
+            if (index === 0) {
+                div.className = 'data-card-tag';
+            } else if (index === 1) {
+                div.className = 'data-card-title';
+            } else if (index === 2) {
+                const hr = document.createElement('hr');
+                hr.style.border = '1px solid lightgrey';
```

Comment:
Handle styles via CSS. Apply a classname via `JS` and let `CSS` pick up the changes.